spec/users_test.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+require 'pry'
+
+require_relative '../lib/ephemeral/app/api.rb'
+
+
+describe "User" do
+  include Rack::Test::Methods
+
+  def app
+    Ephemeral::API    
+  end
+
+  describe "signing up" do
+    before do 
+      post "/v1/users", user: {
```

Comment:
The contents of this before block should actually be in the `it "creates a user" do` block.